rfcs/013-gitlab-spam-checker/013-gitlab-spam-checker.md

```diff
@@ -0,0 +1,93 @@
+# Implementing a Spam Management Service for GitLab
+
+- Start Date: 2023-05-19
+- Status: Proposed
+
+## Summary
+
+This proposal aims to introduce a spam management service to mitigate spam on GitLab and improve user experience and security. This service will use machine learning models to detect and eliminate spam across the platform in real-time, focusing on snippets, issues, notes, projects, and user profiles.
+
+## Problem Statement
+
+GitLab, a service on which RenkuLab relies, is not immune to the issue of spam. Spam can infiltrate various platform aspects, including snippets, issues, notes, projects, and user profiles. Spam not only disrupts the user experience but also poses potential security risks. GitLab lacks a robust, real-time solution to identify and eliminate spam across all attack vectors.
+
+## Key Assumptions
+
+- The proposed machine learning model will be built in consultation with the innovation team. The machine learning model's specifics are out of this RFC's scope.
+
+## Possible Solutions
```

Comment:
I think it would be good here to itemize the options in bullets.

 I also think it would be good to highlight that the problem is quite specific and eg email spam detection mechanisms can not be readily applied as the natuire of the content is quite different...

Reply:
Well, we don't know about that. We might try building on email spam detection models. Its out of the scope of this RFC though.